src/GeneralStateTestsFiller/stMemoryTest/memReturnFiller.json

```diff
@@ -50,7 +50,7 @@
         },
         "transaction" : {
             "data" : [
-                "0xff55883355001144bbccddffeeaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+                ":raw 0xff55883355001144bbccddffeeaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
```

Comment:
this is not raw. this is unknown bytecode. thats why warning here is needed. we need fillers to be more readable

Reply:
This isn't bytecode at all, it's just random bytes that get copied. But I'll remove the `:raw` anyway.